rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/eng.tpl.txt

```diff
@@ -0,0 +1,39 @@
+{{ order.library.name }} - Order [{{ order.reference }}]
+
+Our reference: {{ order.reference }}
+Date : {{ order.order_date }}
+
+Dear Sir,
+
+We would like to order the following list of books:
+
+------------------------------------------------------------
+{%- for line in order.order_lines %}
+Title : {{ line.document.title }}
+{%- if line.document.identifiers %}
+ISBN : {{ line.document.identifiers | join(', ') }}
+{%- endif %}
+{%- if line.note %}
+Note : {{ line.note }}
```

Comment:
filter by `vendor_note`, we do not want to send the `staff_note` to the vendor

Reply:
This is the beauty of custom dumper :-)
into the context i build to render the template, I take only vendor note. So `line.note` and `order.note` are filtered vendor note (check the acq_order dumper and acq_order_line dumper classes)